src/main/java/roomescape/service/TimeService.java

```diff
@@ -0,0 +1,44 @@
+package roomescape.service;
+
+import org.springframework.stereotype.Service;
+import roomescape.dao.TimeDAO;
+import roomescape.domain.Time;
+import roomescape.dto.TimeDto;
+import roomescape.dto.TimeResDto;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class TimeService {
+    private final TimeDAO timeDAO;
+
+    public TimeService(TimeDAO timeDAO) {
+        this.timeDAO = timeDAO;
+    }
+
+    public TimeResDto addTime(TimeDto timeDto) {
+        Time time = new Time(0, timeDto.time());
+        timeDAO.insert(time);
+        int id = timeDAO.getId(time);
+        time.setId(id);
+        return new TimeResDto(id, time.getTime());
+    }
+
+    public List<TimeResDto> getAllTimes() {
+        List<Time> times = timeDAO.findAll();
+        return times.stream()
+                .map(TimeResDto::from)
+                .collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    public List<TimeResDto> getAllTimes() {
        List<Time> times = timeDAO.findAll();
        return times.stream()
                .map(TimeResDto::from)
                .toList();
    }
```

- 저는 toList()가 UnmodifiableList 를 반환하기 때문에 주로 사용해요!
- 준영이 직접 collect(Collectors.toList())와 toList()에 대해서 학습하고 차이를 알려주세요!

Reply:
*  `Collectors.toList()`는 가변 리스트를 반환하지만 `toList()`는 불변 리스트를 반환합니다. 컬렉션을 수정할 필요가 있는 경우 Collectors를 사용하고 불변성을 유지하고 싶을 경우 toList()를 사용한다고 합니다. 결과값을 불변성으로 보장하기 위해  `toList()`를 사용하는 것이 좋아보이는 것 같습니다 감사합니다! 😊